src/lib/utils.ts

```diff
@@ -333,4 +343,4 @@ export function getState(ctx: any, msg: string, incrValue?: string) {
   }
 
   return { state: ctx.state, msg };
-}
+}
```

Comment:
```suggestion
}

```

Adding back newline to end of file